core/src/test/java/org/apache/calcite/test/RelBuilderTest.java

```diff
@@ -1692,6 +1692,40 @@ private RelNode buildRelWithDuplicateAggregates(
     assertThat(root, hasTree(expected));
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4665">[CALCITE-4665]
+   * groupKeys contain unused column.</a>. */
+  @Test void testGroupingSetWithGroupKeysContainUnusedColumn() {
```

Comment:
Containing may be better

Reply:
The comment should be the actual JIRA subject. Which I think should be 'Allow Aggregate.groupSet to contain columns not in any of the groupSets'.

Reply:
done